modules/packages/LinearAlgebra.chpl

```diff
@@ -3450,7 +3450,7 @@ module Sparse {
         end = D.shape(0);
       }
       var indices : [start..end] (D.idxType, D.idxType);
-      forall ind in {start..end} with (ref indices) {
```

Comment:
I wonder if this is a performance improvement. Do we track LinearAlgebra perf anywhere?

Reply:
I don't think we have a lot, if any.

FWIW, I would not expect a major boost here. `{...}` has been shown to inhibit vectorization, but given the loop body here I doubt it matters much